src/test/scala/nl/tradecloud/kafka/KafkaSubscriberSpec.scala

```diff
@@ -105,6 +106,33 @@ class KafkaSubscriberSpec extends TestKit(ActorSystem("KafkaSubscriberSpec", Con
 
       receiverProbe.expectNoMessage(defaultNegativeTimeout)
     }
+
+    "drop messages with an invalid type" in {
+      val receiverProbe = TestProbe("receiver")
+      val publisher = new KafkaPublisher(system)
+
+      val subscriber = new KafkaSubscriber(
+        serviceName = "test",
+        group = "test_group_0",
+        topics = Set("test_topic_5"),
+        system = system,
+        configurationProperties = Seq(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG -> "earliest")
+      )
+
+      subscriber.atLeastOnce[String](
+        Flow[KafkaMessage[String]].map { msg =>
+          receiverProbe.ref ! msg.msg
+
+          msg.offset
```

Comment:
What does  msg.offset do?

Reply:
The flow argument passed to atLeastOnce function, is of type Flow[KafkaMessage, CommittableOffset, _], so it should output the offset of type CommittableOffset